core/code/entity_decode.js

```diff
@@ -85,7 +88,8 @@ function summaryPortalData(a) {
     mission:       a[10],
     mission50plus: a[11],
     artifactBrief: parseArtifactBrief(a[12]),
-    timestamp:     a[13]
+    timestamp:     a[13],
+    history:       a[18]
```

Comment:
It looks like this is actually an optional field, like `mods` or `resonators`. From what I can see, it would be better to define this as part of the detailed portal data below.